src/ServiceControl.Audit/Auditing/AuditPersister.cs

```diff
@@ -43,10 +44,11 @@ public async Task<IReadOnlyList<MessageContext>> Persist(List<MessageContext> co
             }
 
             var storedContexts = new List<MessageContext>(contexts.Count);
+            BulkInsertOperation bulkInsert = null;
```

Comment:
Is it not better to put this in a `using` now?

Reply:
Not really. The bulk insert uses very awkward and evil sync over async which I would rather not want on this path